rotkehlchen/tests/fixtures/blockchain.py

```diff
@@ -186,6 +188,12 @@ def fixture_optimism_mock_data():
     return {}
 
 
+@pytest.fixture(name='polygon_pos_mock_data')
```

Comment:
With the way we have switched to VCR we probably don't need all these. Both for optimism and polygon.

Reply:
Hmm but I see it still being used in certain ethereum tests. Maybe we could handle this change later in a separate PR since it's not something polygon-specific?

Reply:
yes can definitely move later

Reply:
it is used in some tests, yes but it's all old tests before vcr. No new tests like that should be written. So you definitely don't need to add these fixtures for polygon